ios/FluentUI.Demo/FluentUI.Demo/Demos/ListDemoController.swift

```diff
@@ -132,6 +88,36 @@ class ListDemoController: DemoController {
                                      demoControllerView.safeAreaLayoutGuide.trailingAnchor.constraint(equalTo: listView.trailingAnchor)])
     }
 
+    struct PersonaDataNode {
+        var personaData: PersonaData
+        var children: [PersonaDataNode] = []
+        var isExpanded: Bool = true
+    }
+
+    private func createSamplePersonaCell(cellState: MSFListCellState, personaDataNode: PersonaDataNode) {
+        let personaData = personaDataNode.personaData
+        let personaChildren = personaDataNode.children
+        let avatar = createAvatarView(size: .medium,
+                                      name: personaData.name,
+                                      image: personaData.image,
+                                      style: .default)
+        cellState.title = avatar.state.primaryText ?? ""
+        cellState.leadingUIView = avatar.view
+        cellState.isExpanded = personaDataNode.isExpanded
+        cellState.hasDivider = true
+
+        guard !personaChildren.isEmpty else {
```

Comment:
nit: It's not a super objective criterion, but I don't think this qualifies as early return because the creation of the avatar is already done at this point.

I think just a simple ternary operator like below should suffice:

```
        cellState.onTapAction = personaChildren.isEmpty ? {
            self.showAlertForAvatarTapped(name: personaData.name)
        } : nil
```